rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -182,11 +181,15 @@ class BakeryController {
         }
       }
 
-      def packerCommand = cloudProviderBakeHandler.producePackerCommand(region, bakeRequest)
-      def jobRequest = new JobRequest(tokenizedCommand: packerCommand, maskedPackerParameters: cloudProviderBakeHandler.maskedPackerParameters, jobId: bakeRequest.request_id)
+      def bakeRecipe = cloudProviderBakeHandler.produceBakeRecipe(region, bakeRequest)
+      def jobRequest = new JobRequest (
```

Comment:
Yeah, I'm not a fan of the all-the-way-to-the-right indentations, so I was launching a silent rebellion in this PR to check the response. But consistency is even more important so I'll go with the flow on this one. No biggy.